test/iron-location.html

```diff
@@ -436,7 +449,6 @@
       });
 
       test('of the query portion of the URL', function() {
-        expect(window.location.search).to.be.equal('');
```

Comment:
Then shouldn't [line 441](https://github.com/PolymerElements/iron-location/pull/54/files/fa0677a716039c30202e2eeb4042b7af0ccd9c83#diff-9c2bf7820c8af6945a0c10b98a7299b4L441) fail? Or `iron-location` overrides the previous flags?

In any case, you could still test this:
```js
expect(window.location.search.replace('?wc-shadydom=true&wc-ce=true', '')).to.be.equal('');